docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -23,21 +25,95 @@ If the tested page does not use HTTPS-specific features (like
+4. Create a config file `testdomain.ext` with the following contents:
+  
+    ```sh
+    authorityKeyIdentifier=keyid,issuer
+    basicConstraints=CA:FALSE
+    keyUsage = digitalSignature, nonRepudiation, keyEncipherment, dataEncipherment
+    subjectAltName = @alt_names
+
+    [alt_names]
+    DNS.1 = localhost
+    ```
+
+5. Now create the certificate using our `.csr`, the CA private key (`myCA.key`), the CA certificate (`myCA.pem`), and a config file:
+
+    ```sh
+    openssl x509 -req -in testingdomain.csr -CA myCA.pem -CAkey myCA.key -CAcreateserial -out testingdomain.crt -days 825 -sha256 -extfile testdomain.ext
+    ```
+
+6. Finally, export the certificate to PKCS#12 `.pfx` format:
+
+    ```sh
+    openssl pkcs12 -export -out testingdomain.pfx -inkey testingdomain.key -in testingdomain.crt -certfile myCA.pem
+    ```
+
+In **PowerShell**, [install OpenSSL](https://adamtheautomator.com/openssl-powershell/#Installing_OpenSSL_using_Chocolatey) and [update environment variables](https://adamtheautomator.com/openssl-powershell/#Update_PowerShell_Profile_Environment_Variables). Afterwards, proceed with the instructions [above](#generate-a-certificate-from-the-command-line).
+
+##### Run Tests
 
 When you run tests from the command line, use the [--ssl](../../reference/command-line-interface.md#--ssl-options) flag to enable HTTPS on a proxy server. Specify options required to initialize [a Node.js HTTPS server](https://nodejs.org/api/https.html#https_https_createserver_options_requestlistener) after this flag in a semicolon-separated string. The most commonly used SSL options are described in the [TLS topic](https://nodejs.org/api/tls.html#tls_tls_createsecurecontext_options) in the Node.js documentation.
 
-The example below uses the PFX encoded private key and certificate chain to create an HTTPS server.
+The examples below use a previously generated `.pfx` file to establish an HTTPS server.
+
+In ***Chrome*** and ***Chromium-Based*** Browsers use `--allow-insecure-localhost` flag and [`--hostname`](../../reference/command-line-interface.md#--hostname-name) CLI parameter to make Chrome trust the certificate.
+
+```sh
+testcafe chrome --allow-insecure-localhost --hostname localhost test.js --ssl pfx=path/to/certificate.pfx;rejectUnauthorized=true;
+```
+
+In ***Firefox*** it is necessary to add your self-signed root authority to the trusted list. Follow the steps:
+
+1. [Create a new profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles#w_creating-a-profile) in Firefox settings;
+2. [Run Firefox with this profile](https://support.mozilla.org/en-US/kb/profile-manager-create-remove-switch-firefox-profiles#w_manage-profiles-when-firefox-is-open) and [import the CA certificate](https://support.mozilla.org/en-US/questions/1059377) (`myCA.pem`) you generated previously. Make sure to check the **"This certificate can identify websites"** checkbox;
```

Comment:
`**"This certificate can identify websites"**`

We don't need both bold and quotes, just bold is enough to mention a UI option:

`**This certificate can identify websites**`

Reply:
**Make sure to check** can be read like **Don't forget to pay your attention to**, like in 'make sure to check this movie'

So, I would say 'Ensure the **This certificate can identify websites** checkbox is checked.'